### DISCUSSION TITLE
Open Router

### DISCUSSION THREAD

**Discussion by Jblckwd** [n4ze3m/page-assist#232] [Q&A]
Trying to use llama 3.2 via open router in custom mode and for some reason it's not working and doesn't fetch the model after configuration from custom mode screen.

**Comment by n4ze3m** [comment]
Hey, sorry about the issue. Did you add it like this?

https://github.com/user-attachments/assets/ef77b498-50c6-4f53-acd5-1f72f1d86bfb

**Reply by Jblckwd** [reply]
Ahhh I missed the area where you can choose open router. I had just tried using custom. All is working as expected. Amazing work
